src/NBitcoin/ProvenBlockHeader.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+
+namespace NBitcoin
+{
+    /// <summary>
+    /// Extension to an existing <see cref="BlockHeader"/> which is used in PoS to prevent attacker from constructing
+    /// a fake chain of headers that has more work than the valid chain and attacking a node.
+    ///
```

Comment:
Use `<para>...</para>` instead of empty line

Reply:
Done